fastestimator/dataset/data/medmnist.py

```diff
@@ -0,0 +1,83 @@
+# Copyright 2022 The FastEstimator Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+import os
+from pathlib import Path
+from typing import Tuple
+
+import numpy as np
+
+from fastestimator.dataset.numpy_dataset import NumpyDataset
+from fastestimator.util.google_download_util import _download_file_from_google_drive
+
+dataset_ids = {
+    "chestmnist": "1lGZtFjlviRf_vwmxB4NgApjCdfDuh8pk",
+    "adrenalmnist3d": "1E__zT_PqXlaiyW3r6Ii98UVVv5-nDZ_q",
+    "bloodmnist": "17r9NENUnUoqnWZ-zJaveLU40R746A-ul",
+    "breastmnist": "1E3fWybVlOXSIjg_kFSQ-nwOikLgrbsJc",
+    "dermamnist": "1k-vw5otV5rCQLVQKyxmo1mhI3KqDbwU5",
+    "fracturemnist3d": "104OYiONKAsN2O-VeVGS7ReXZI4GtNsZk",
+    "nodulemnist3d": "1krf_Uv-bA5CEt_dCD6h8xJbzZk9ETWLU",
+    "octmnist": "1yRhEBDQwHu5jSnrzIm87Cazns3Eesv4b",
+    "organamnist": "1yRhEBDQwHu5jSnrzIm87Cazns3Eesv4b",
+    "organcmnist": "1GqXyaV6arJq0O-3Fn2hvBw77MG9fHeoz",
+    "organmnist3d": "1RQiHLj35u3m6GCiBKJFnb9VH76u7yRi0",
+    "organsmnist": "1spWIVxKaLvWAxLHGLSr0IJJBSm0MDBB4",
+    "pathmnist": "12He-AovBA5ReIg1snhA6Jm2jQCvKKYp7",
+    "pneumoniamnist": "1ObI6UzRYZby9YCmk53m1WCXo747wzNqT",
+    "retinamnist": "1dugDJx_Z9nvtbD9SidxkVjgvysA93IG7",
+    "synapsemnist3d": "1JCBppD-bCYQhjwufJ63OS8XgMaT3sRv5",
+    "tissuemnist": "1M5wec6b-iiLPjVs3MrCrAlKCxwjuEL2z",
+    "vesselmnist3d": "1orrps7d-SKB4kFPV0jaN3r_jfYpDtGpP",
+}
+
+
+def load_data(
+    root_dir: str = None,
+    dataset_name: str = None,
```

Comment:
add docstring, and in the dataset_name, let user know of the options.  Also, put the original dataset link in the general description, in case people want to check more details.